renovate.json

```diff
@@ -2,20 +2,48 @@
   "commitMessageAction": "update",
   "commitMessagePrefix": "⬆️(dependencies)",
   "commitMessageTopic": "{{depName}}",
-  "enabledManagers": ["npm"],
+  "enabledManagers": ["npm", "setup-cfg"],
   "extends": ["config:base"],
+  "labels": ["dependencies"],
+  "prConcurrentLimit": 2,
+  "prCreation": "not-pending",
+  "prHourlyLimit": 1,
+  "rebaseWhen": "behind-base-branch",
+  "semanticCommits": "disabled",
+  "separateMajorMinor": false,
+  "updateNotScheduled": true,
   "packageRules": [
     {
-      "packagePatterns": ["*"],
       "groupName": "npm dependencies",
-      "schedule": ["before 7am on monday"]
+      "matchManagers": ["npm"],
+      "schedule": ["before 7am on monday"],
+      "matchPackagePatterns": ["*"]
+    },
+    {
+      "groupName": "python dependencies",
+      "matchManagers": ["setup-cfg"],
+      "schedule": ["before 7am on monday"],
+      "matchPackagePatterns": ["*"],
+      "excludePackageNames": [
+        "arrow",
+        "Django",
+        "dj-pagination",
+        "django-cms",
+        "django-parler",
+        "djangocms-file",
+        "djangocms-googlemap",
+        "djangocms-link",
+        "djangocms-picture",
+        "djangocms-text-ckeditor",
+        "djangocms-video",
+        "djangorestframework",
+        "elasticsearch",
+        "social-auth-core[openidconnect]",
+        "social-auth-app-django",
+        "django-redis",
```

Comment:
You decided to exclude both `django-redis` and `exrex`. Is it intended ? They should be excluded and they were not with pyup ?

Reply:
Oh copy/pasta failed !